### PR TITLE
Make RSSI a sint32

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

File: src/service/packet_router.proto [hunk 1]
```diff
@@ -8,7 +8,7 @@ import "data_rate.proto";
 message packet_router_packet_up_v1 {
   bytes payload = 1;
   uint64 timestamp = 2;
-  uint32 rssi = 3;
+  sint32 rssi = 3;
```
Comment (dpezely) [inline src/service/packet_router.proto:11]: Add comment indicating units:
```suggestion
  // LoRa signal strength in dB.
  sint32 rssi = 3;
```

File: src/service/packet_router.proto [hunk 2]
```diff
@@ -24,7 +24,7 @@ message packet_router_packet_report_v1 {
   uint32 oui = 2;
   uint64 net_id = 3;
   // LoRa signal strength
```
Comment (dpezely) [inline src/service/packet_router.proto:26]: Add units:
```suggestion
  // LoRa signal strength in dB.
```

**Review by dpezely — APPROVED** [review]
Add comments that include units, per various discussions over this past summer.

File: src/service/packet_router.proto (hunk 1 above)
Comment (madninja) [reply src/service/packet_router.proto:11]: should this be LoRa Signal Strength in `ddbi`?

Comment (michaeldjeffrey) [reply src/service/packet_router.proto:11]: semtech protocol marks it as `dBm`(?)
https://github.com/Lora-net/packet_forwarder/blob/master/PROTOCOL.TXT#L144

**Pull request merged**